charts/kube-master/templates/api.yaml

```diff
@@ -165,6 +165,15 @@ spec:
             - mountPath: /etc/kubernetes/bootstrap
               name: bootstrap
               readOnly: true
+          livenessProbe:
+            httpGet:
+              path: /healthz
```

Comment:
```suggestion
              path: /healthz/etcd
```

Reply:
I think `healthz` aggregates various health checks, we can restrict killing the pod to when the etcd endpoint check is failing specifically.